engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicStateStatus.scala

```diff
@@ -11,7 +11,7 @@ import java.time.format.DateTimeFormatter
 object PeriodicStateStatus {
 
   //without seconds because we do not deploy with that precision
-  private val format = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")
+  val format: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")
```

Comment:
Format -> it's a constant, hence it should start from big letter.

Reply:
Fixed.